sdk/core/System.BinaryData/src/TaskExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace System
```

Comment:
Why a copy?

Reply:
I figured I would only include the method that was needed here considering that this will be moving over to dotnet/runtime.

Reply:
Removing this as per a different comment.